### PR TITLE
Docs: Unify the styles usage in the Angular guide.

### PR DESCRIPTION
**Pull request by gorzelinski**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Docs: Unify the styles usage in the Angular guide.

---

### Additional information

_For example – encountered issues, assumptions you had to make, other affected tickets, etc._

### CODE REVIEW

File: docs/getting-started/integrations/angular.md [hunk 1]
```diff
@@ -125,46 +125,56 @@ Then, install the [CKEditor&nbsp;5 WYSIWYG editor component for Angular](https:/
...
 }
 ```
 
+Depending on the plugins you used, you may need to import the first or both CSS files. Angular, by default, scope styles to a particular component. Because of that, the editor may not detect attached styles. You must set the encapsulation option to `ViewEncapsulation.None` to turn this scoping off.
+
+```css
+/* app.component.css */
+
+@import 'ckeditor5/ckeditor5.css';
+@import 'ckeditor5-premium-features/ckeditor5-premium-features.css';
```
Comment (gorzelinski) [inline docs/getting-started/integrations/angular.md:175]: These imports work with or without a tilde (~), so I omitted it.

File: docs/getting-started/integrations/angular.md [hunk 2]
```diff
@@ -205,35 +215,42 @@ Then, import the editor in your Angular component and assign it to a `public` pr
...
+    title = 'angular';
+
+    public Editor = ClassicEditor;
+    public config = {
+        toolbar: [ 'undo', 'redo', '|', 'bold', 'italic' ],
+        plugins: [
+            Bold, Essentials, Italic, Mention, Paragraph, SlashCommand, Undo
+        ],
+        licenseKey: '<YOUR_LICENSE_KEY>',
+        // mention: {
```
Comment (gorzelinski) [inline docs/getting-started/integrations/angular.md:238]: The empty mention configuration option causes problems with types. I put it in the comment.

**Comment by Witoso** [comment]
This was tested against plain Angular 16 (default) and Angular 17 (default), right?

File: docs/getting-started/integrations/angular.md [hunk 3]
```diff
@@ -227,13 +235,22 @@ export class AppComponent {
 			Bold, Essentials, Italic, Mention, Paragraph, SlashCommand, Undo
 		],
 		licenseKey: '<YOUR_LICENSE_KEY>',
-		mention: {
-			// Mention configuration
-		}
+		// mention: {
+		//     // Mention configuration
```
Comment (godai78) [inline docs/getting-started/integrations/angular.md:239]: This looks somewhat weir, the double comment. Wouldn't one be enough?

**Review by godai78 — APPROVED** [review]

**Comment by gorzelinski** [comment]
> This was tested against plain Angular 16 (default) and Angular 17 (default), right?

Yes, I set up projects in those versions.

File: docs/getting-started/integrations/angular.md (hunk 3 above)
Comment (gorzelinski) [reply docs/getting-started/integrations/angular.md:239]: Right, it is excessive - a single comment is enough.

**Review by mabryl — APPROVED** [review]
LGTM as well :)

**Comment by Witoso** [comment]
cc Builder team @ckeditor/ckeditor-5-devops

**Pull request merged**